.github/workflows/unittests.yml

```diff
@@ -64,90 +64,73 @@ jobs:
-      - run: sudo apt-get install -y maven
-      - run: sudo apt-get install -y libboost-all-dev libcurl4-openssl-dev
-      - run: sudo apt-get install -y libssl-dev
-      - run: sudo apt-get install -y libz-dev
-      - run: sudo apt-get install -y llvm clang
-      - run: sudo apt-get install -y libdwarf-dev
-      - run: sudo apt-get install -y libre2-dev
-      #  no need to cache these as it's running on self host server
-      #- name: ccache cache files
-      #  uses: actions/cache@v2
-      #  with:
-      #    path: ~/.ccache
-      #    key: ${{ github.job }}-ccache
-      #    restore-keys: |
-      #      ${{ github.job }}-ccache
-      #- name: Cache local Maven repository
-      #  uses: actions/cache@v2
-      #  with:
-      #    path: ~/.m2/repository
-      #    key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-      #    restore-keys: |
-      #      ${{ runner.os }}-maven-
-      - name: Install Googletest
+      - name: Checkout gluten-te code
         run: |
-          sudo apt-get install -y libgtest-dev
-          cd /usr/src/gtest
-          sudo cmake CMakeLists.txt -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=/usr/local
-          sudo make -j$(nproc)
-          sudo apt-get install -y google-mock
+          git clone -b main https://github.com/zhztheplayer/gluten-te.git gluten-te
       - name: Install Gluten Velox-backend spark3.2
+        env:
+          HTTP_PROXY_HOST: ${{ secrets.SR404_PROXY_HOST }}
+          HTTP_PROXY_PORT: ${{ secrets.SR404_PROXY_PORT }}
         run: |
-          rm -rf ~/.m2/repository/org/apache/arrow && rm -rf ~/.m2/repository/io/glutenproject/
-          mvn clean install -Pspark-3.2 -DskipTests -Pbackends-velox -Dbuild_protobuf=OFF -Dbuild_cpp=ON -Dbuild_velox=ON -Dbuild_velox_from_source=ON -Dbuild_gazelle_cpp=OFF -Dbuild_arrow=ON -Dcpp_tests=ON -Dcpp_benchmarks=ON
+          gluten-te/cmvn.sh clean install -Pspark-3.2 -DskipTests -Pbackends-velox -Dbuild_protobuf=OFF -Dbuild_cpp=ON -Dbuild_velox=ON -Dbuild_velox_from_source=ON -Dbuild_gazelle_cpp=OFF -Dbuild_arrow=ON -Dcpp_tests=ON -Dcpp_benchmarks=ON
       - name: Run unit tests
+        env:
+          HTTP_PROXY_HOST: ${{ secrets.SR404_PROXY_HOST }}
+          HTTP_PROXY_PORT: ${{ secrets.SR404_PROXY_PORT }}
         run: |
-          mvn test -Pspark-3.2 -DtagsToExclude=io.glutenproject.tags.SkipTestTags -Dcheckstyle.skip -Dexec.skip -Pbackends-velox -Dbackend_type=velox
-          cd cpp/
-          mkdir -p build
-          cd build
-          cmake .. -DBUILD_ARROW=0 -DTESTS=1
-          make -j$(nproc)
-          # ctest -V
+          gluten-te/cmvn.sh test -Pspark-3.2 -DtagsToExclude=io.glutenproject.tags.SkipTestTags -Dcheckstyle.skip -Dexec.skip -Pbackends-velox -Dbackend_type=velox
+          # cd cpp/
+          # mkdir -p build
+          # cd build
+          # cmake .. -DBUILD_ARROW=0 -DTESTS=1
+          # make -j$(nproc)
+          ## ctest -V
```

Comment:
Should we remove this?

Reply:
I think we can remove it,` -Dcpp_tests=ON` is enough